### PR TITLE
Add !tellmewhy command

### PR DESCRIPTION
**Pull request by pascuale2**

### CODE REVIEW

File: Commands/trashtellmewhy.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    soundPaths = [
+        "Assets/heartache.m4a",
+        "Assets/mistake.m4a",
+        "Assets/hearyousay.m4a",
+    ];
+
+    run(client, message, args) {
+        const player = require('../Utility/playvoice');
+        const randomIndex = Math.floor(Math.random() * this.soundPaths.length);
+        player.Play(message, voice.createAudioResource(this.soundPaths[randomIndex]));
```
Comment (veremenko-y) [inline Commands/trashtellmewhy.js]: It's going to play duplicates. fix it! :trash: :trash: :trash:

Comment (veremenko-y) [reply Commands/trashtellmewhy.js]: All you need to do is something like this

```
class TrashTellMeWhy{
	lastIndex = 0
    run(client, message, args) {
        const player = require('../Utility/playvoice');
		do {
			const randomIndex = Math.floor(Math.random() * this.soundPaths.length);
			if(randomIndex != this.lastIndex) {
				this.lastIndex = randomIndex;
			}
		} while (soundPaths.length > 1);
        player.Play(message, voice.createAudioResource(this.soundPaths[this.lastIndex]));
```

**Comment by veremenko-y** [comment]
test

File: Commands/trashtellmewhy.js (hunk 1 above)
Comment (veremenko-y) [reply Commands/trashtellmewhy.js]: SUTMIIIIIT

**Comment by veremenko-y** [comment]
TRAASH

**Review by veremenko-y — CHANGES_REQUESTED** [review]
TRAAASH

File: Commands/trashtellmewhy.js (hunk 1 above)
Comment (pascuale2) [reply Commands/trashtellmewhy.js]: Should be fixed in 4aecbaee1ceb76d8158ae61f6c4ea9e3fba79faa

**Pull request merged**